addons/website_slides/tests/test_ui_wslides.py

```diff
@@ -224,6 +224,10 @@ def _add_reaction(self, message, reaction):
             },
         )
 
+    def test_course_review_modification(self):
+        self.user_portal.karma = 20
```

Comment:
Give the portal user enough karma so they don't have to go through the course to post a review (avoid unnecessary steps).